docs/how-to-guides/how-to-create-backup.md

```diff
@@ -1,33 +1,33 @@
 ---
 id: how-to-create-backup
-title: How to create backup
+title: Create backup
 sidebar_label: How to create backup
 ---
 
-While running, Memgraph generates several different files in its data
-directory. This a location where Memgraph saves all permanent data. The default
-data directory is `/var/lib/memgraph` and you can change it in the main
-[Memgraph configuration file](/docs/memgraph/reference-guide/configuration).
+While running, Memgraph generates several different files in the data directory
+it saves all permanent data. The default data directory is `/var/lib/memgraph`
+and you can change it in the main [Memgraph configuration
+file](/docs/memgraph/reference-guide/configuration).
 
-The data directory includes multiple different subdirectories, one of them being
+The data directory includes multiple different subdirectories, one of them is
 the storage directory, which contains the durability files. In that directory,
 Memgraph periodically generates [snapshots and WAL
 files](/docs/memgraph/under-the-hood/storage) that contain Memgraph's data in a
 recoverable format.
 
-Snapshot is created periodically based on the time defined with the
-`--storage-snapshot-interval-sec` config in the Memgraph configuration file.
+Snapshots are created periodically based on the time defined by the
+`--storage-snapshot-interval-sec` flag in the Memgraph configuration file.
 
```

Comment:
I would add here a link to how-to tutorial about changing Memgraph configuration:
/how-to-guides/config-logs